src/pages/Login.tsx

```diff
@@ -0,0 +1,23 @@
+import AppBackButton from "../components/atom/AppBackButton";
+import LoginForm from "../components/molecules/LoginForm";
+
+const Login = () => {
+    return (
+        <div className="h-screen flex flex-col page-bg text-blue-dark p-8 justify-between">
```

Comment:
It would be nice to have parametric color scheme tied to scss, so at some point someone could go to scss and change the main colors, including text, backgrounds (imagine we now want to change all accent text or notification text color to a black).
In that case we could have here a reference to "text-main-color", "text-info-color" instead to a text-blue-dark color for example which may require refactor in the future.